var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -169,6 +173,11 @@ def configure(self):
     def configure(self):
         configure('-fast',
                   '-no-webkit',
+                  # XXX(osx): sdk stuff; should depend on the architecture and the SDKs available.
+                  '-arch', 'x86_64',
+                  '-sdk', '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk',
```

Comment:
OK, so the homebrew patch isn't enough. It doesn't even configure against 10.11. 10.9 still works; I'll add a comment about the caveat, but this gets most of the work out of the way for getting 10.11 to work.